handlers.go

```diff
@@ -135,3 +136,31 @@ func (c channelHandler) Handle(ctx context.Context, call *InboundCall) {
 func (c channelHandler) Register(h Handler, methodName string) {
 	c.ch.GetSubChannel(c.ch.PeerInfo().ServiceName).Register(h, methodName)
 }
+
+// userHandlerWithIgnore is a Handler that wraps a localHandler backed by the channel.
+// and a user provided handler.
+// The inbound call will be handled by user handler, unless the call's
+// service and method name are configured to be handled by localHandler
+// from ignore.
+type userHandlerWithIgnore struct {
+	localHandler      channelHandler
+	ignoreUserHandler map[string]struct{} // key is serviceName::method format
+	userHandler       Handler
+}
+
+func (u userHandlerWithIgnore) Handle(ctx context.Context, call *InboundCall) {
+	var sb strings.Builder
+	sb.WriteString(call.ServiceName())
+	sb.WriteString("::")
+	sb.Write(call.Method())
+
+	if _, ok := u.ignoreUserHandler[sb.String()]; ok {
```

Comment:
see https://github.com/uber/tchannel-go/pull/831#discussion_r681340445

To avoid this, you could instead have `map[string]map[string]struct{}` (first map for service name, second for method name)